distro/src/avail/Avail.avail/Foundation.avail/Tuples.avail

```diff
@@ -3243,18 +3344,36 @@ Semantic restriction "_'s⁇last" is
  */
 Public stable method "∑_" is
 [
-	intTuple : extended integer*
+	intTuple : extended integer+
 |
-	0
+	left fold intTuple through
+		[i : extended integer, ri : extended integer | i + ri]
 ] : extended integer;
 
+/**
+ * Sum the elements of the specified tuple of numbers.
+ *
+ * @method "∑_"
+ * @param "numTuple" "number*"
+ *        A tuple of numbers.
+ * @returns "number"
+ *          A number.
+ * @category "Tuples" "Mathematics"
+ */
+Public stable method "∑_" is
+[
+	numTuple : number+
```

Comment:
You'll want to make this work for empty tuples also, in a similar way to the way that I described above.